example/src/App.tsx

```diff
@@ -1004,7 +1004,7 @@ class App extends React.Component<{}, State> {
             active={this.state.outterDrawer}
             activeContentId={this.state.outterDrawerId}
             mode="slide"
-            componentWidth="large"
+            componentWidth="530px"
             componentLabel={this.state.outterDrawerLabel}
```

Comment:
Can we make '530'as a constant, and use it that way.

Reply:
@ankitsaraogi user can pass value in px or em or rem, that's why I have kept it like that